app/lib/Email.scala

```diff
@@ -50,14 +50,7 @@ case class Email(
     headers.foreach(t => msg.addHeader(t._1, t._2))
     addresses.replyTo.foreach(r => msg.setReplyTo(Array(new InternetAddress(r))))
 
-    // Add a MIME part to the message
-    val mp = new MimeMultipart()
-
-    val part = new MimeBodyPart()
-    part.setContent(bodyText, "text/plain")
-    mp.addBodyPart(part)
-
-    msg.setContent(mp)
+    msg.setContent(bodyText, "text/plain")
```

Comment:
This probably works, but I prefer the `setText()` method in the original example you gave. I've added it as #19, but thank you many times over for reading the docs with a clear head, which I clearly was not able to do!